src/lib.rs

```diff
@@ -230,6 +231,19 @@ pub enum ConfigLevel {
     Highest,
 }
 
+/// An enumeration for favoring sides to resolve merge conflicts
+#[derive(PartialEq, Eq, Debug, Copy, Clone)]
+pub enum FileFavor {
+    /// Record conflicts in the index without favoring sides
```

Comment:
Yes I'll change it to copy the docs from libgit2.  I was under the impression that other docs in this code base seemed to be paraphrasing the libgit2 docs but I like copying them.